spec/system/notifications/notifications_page_spec.rb

```diff
@@ -25,35 +25,6 @@ def validate_reply(id)
     expect(page).not_to have_css("span#notifications-number", text: "1")
   end
 
-  xit "allows user to interact with replies" do
-    sidekiq_perform_enqueued_jobs do
-      article = create(:article, user: alex)
-      comment = create(:comment, commentable: article, user: alex)
-      reply = create(:comment, commentable: article, user: leslie, parent: comment)
-      Notification.send_new_comment_notifications_without_delay(reply)
-    end
-
-    visit "/notifications"
-
-    expect(page).to have_css("div.spec-notification")
-    click_button("heart")
-
-    expect(page).to have_css(".reacted")
-
-    click_link("Reply")
-
-    validate_reply(leslie.comments.first.id)
-  end
-
-  it "allows user to follow other users back" do
```

Comment:
It's not new in this PR, but we already have a Cypress test for this in `cypress/integration/seededFlows/notificationsFlows/followUserFromNotifications.spec.js`